### PR TITLE
[RM-17612] Support oracle linux server

### PR DESCRIPTION
**Pull request by alexei38**
I need support oracle linux server.
I use oracle linux server for use support oracle software.
https://en.wikipedia.org/wiki/Oracle_Linux

### CODE REVIEW

**Comment by liewegas** [comment]
test this please

**Comment by alfredodeza** [comment]
ok to test

File: ceph_deploy/hosts/__init__.py [hunk 1]
```diff
@@ -54,7 +54,7 @@ def get(hostname,
         if 'already closed' in getattr(error, 'message', ''):
             raise RuntimeError('remote connection got closed, ensure ``requiretty`` is disabled for %s' % hostname)
     distro_name, release, codename = conn.remote_module.platform_information()
-    if not codename or not _get_distro(distro_name):
```
Comment (alfredodeza) [inline ceph_deploy/hosts/__init__.py]: not checking for `codename` is going to cause issues for other non-oracle targets. Maybe a one-off check for oracle might be better?

Comment (alexei38) [inline ceph_deploy/hosts/__init__.py]: An error in the log jenkins not related with this commit

**Comment by alexei38** [comment]
recheck this commit please

**Comment by alfredodeza** [comment]
test this please

**Comment by alfredodeza** [comment]
@alexei38 this looks good to me. Would you mind going over the contributing doc and following the workflow? https://github.com/ceph/ceph-deploy/blob/master/CONTRIBUTING.rst#contributing-to-ceph-deploy

You will need to create an issue in the tracker and then reference it as part of the commit message. Thanks!

**Pull request merged**